addons/interaction/functions/fnc_manageAce3Interactions.sqf

```diff
@@ -0,0 +1,96 @@
+/**
+ * Receives a status update and sets ACE3 dragging, carrying and cargo accordingly.
+ *
+ * Arguments:
+ * 0: Equipment <OBJECT>
+ * 1: Condition <STRING>
+ * 2: Status <BOOLEAN>
+ *
+ * Returns:
+ * none
+ *
+ * Example:
+ * [_computer, "powerConnected", true] call AE3_interaction_fnc_manageAce3Interactions
+ */
+
+params ["_target", "_condition", "_status"];
+
+// ace_dragging_fnc_setCarryable and ace_dragging_fnc_setDraggable need to be executed on all machines with JIP
+// ace_cargo_fnc_setSize can be executed locally because it handles JIP and remoteExecCall on it's own
+
+//this function only runs once per triggered interaction by player on players machine
+private _settingsAce3 = _target getVariable "AE3_SettingsACE3";
+if (!isNil "_settingsAce3") then
```

Comment:
Use of exitWith with negated condition would make the function cleaner.

Reply:
Will do that.